From 675a7f1d999f75364a75966511ef0a5bca8ef158 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Thu, 28 Sep 2017 16:22:21 -0500
Subject: [PATCH 251/278] drivers/net: Remove Reference to
 CONFIG_NET_VENDOR_LSI

The correct build for Axxia 5500 is CONFIG_LSI_NET.
CONFIG_NET_VENDOR_LSI is not used.

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 drivers/net/ethernet/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/ethernet/Makefile b/drivers/net/ethernet/Makefile
index fccc86b..549abd8 100644
--- a/drivers/net/ethernet/Makefile
+++ b/drivers/net/ethernet/Makefile
@@ -48,7 +48,6 @@ obj-$(CONFIG_NET_VENDOR_XSCALE) += xscale/
 obj-$(CONFIG_JME) += jme.o
 obj-$(CONFIG_KORINA) += korina.o
 obj-$(CONFIG_LANTIQ_ETOP) += lantiq_etop.o
-obj-$(CONFIG_NET_VENDOR_LSI) += lsi/
 obj-$(CONFIG_NET_VENDOR_MARVELL) += marvell/
 obj-$(CONFIG_NET_VENDOR_MEDIATEK) += mediatek/
 obj-$(CONFIG_NET_VENDOR_MELLANOX) += mellanox/
-- 
1.8.5.2.233.g932f7e4

